src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -477,6 +477,7 @@ ReactDOMComponent.Mixin = {
         ReactDOMInput.mountWrapper(this, props, nativeParent);
         props = ReactDOMInput.getNativeProps(this, props);
         transaction.getReactMountReady().enqueue(trapBubbledEventsLocal, this);
+        transaction.getReactMountReady().enqueue(ReactDOMInput.initializeIndeterminate, this);
```

Comment:
this could be theoretically handled more generally, by having something loop through props propertyInfo and checking for IDL only attributes and setting them. Perhaps that would be more efficient? My assumption is that the above will properly batch everything anyway so it may not matter.